src/overlays/effects/ovl_Effect_Ss_Hahen/z_eff_ss_hahen.c

```diff
@@ -27,8 +27,8 @@ EffectSsInit Effect_Ss_Hahen_InitVars = {
 };
 
 void EffectSsHahen_CheckForObject(EffectSs* this, PlayState* play) {
-    if (((this->rObjBankIndex = Object_GetSlot(&play->objectCtx, this->rObjId)) < 0) ||
-        !Object_IsLoaded(&play->objectCtx, this->rObjBankIndex)) {
+    if (((this->rObjectSlot = Object_GetSlot(&play->objectCtx, this->rObjectId)) < 0) ||
```

Comment:
Looks like you missed these weird effect conditionals that set the variable in the conditional.

Reply:
Found 9 of these. Thanks!